src/recipes/FactoryRecipeId.ts

```diff
@@ -1,2 +1,310 @@
wder_1_C' | 'Recipe_Alternate_ElectromagneticControlRod_1_C' | 'Recipe_Alternate_CrystalOscillator_C' | 'Recipe_Alternate_Concrete_C' | 'Recipe_Alternate_Computer_2_C' | 'Recipe_Alternate_Computer_1_C' | 'Recipe_Alternate_CircuitBoard_2_C' | 'Recipe_Alternate_CircuitBoard_1_C' | 'Recipe_Alternate_Cable_2_C' | 'Recipe_Alternate_Cable_1_C' | 'Recipe_Ficsonium_C' | 'Recipe_FicsoniumFuelRod_C' | 'Recipe_SingularityCell_C' | 'Recipe_SpaceElevatorPart_11_C' | 'Recipe_FilterHazmat_C' | 'Recipe_Quickwire_C' | 'Recipe_Biofuel_C' | 'Recipe_Protein_Hog_C' | 'Recipe_Protein_Spitter_C' | 'Recipe_Biomass_Mycelia_C' | 'Recipe_PowerCrystalShard_1_C' | 'Recipe_Gunpowder_C' | 'Recipe_AlienPowerFuel_C' | 'Recipe_Protein_Stinger_C' | 'Recipe_Protein_Crab_C' | 'Recipe_AlienDNACapsule_C' | 'Recipe_Biomass_AlienProtein_C' | 'Recipe_SpikedRebar_C' | 'Recipe_CartridgeSmart_C' | 'Recipe_Rebar_Stunshot_C' | 'Recipe_FilterGasMask_C' | 'Recipe_NobeliskGas_C' | 'Recipe_Alternate_PolyesterFabric_C' | 'Recipe_Fabric_C' | 'Recipe_SyntheticPowerShard_C' | 'Recipe_PowerCrystalShard_3_C' | 'Recipe_PowerCrystalShard_2_C' | 'Recipe_NobeliskShockwave_C' | 'Recipe_Rebar_Spreadshot_C' | 'Recipe_CartridgeChaos_Packaged_C' | 'Recipe_CartridgeChaos_C' | 'Recipe_NobeliskNuke_C' | 'Recipe_Cartridge_C' | 'Recipe_Rebar_Explosive_C' | 'Recipe_NobeliskCluster_C' | 'Recipe_Nobelisk_C' | 'Recipe_GunpowderMK2_C' | 'Recipe_Snowball_C' | 'Recipe_XmasStar_C' | 'Recipe_XmasWreath_C' | 'Recipe_XmasBallCluster_C' | 'Recipe_XmasBall1_C' | 'Recipe_XmasBall2_C' | 'Recipe_XmasBall3_C' | 'Recipe_XmasBall4_C' | 'Recipe_Snow_C' | 'Recipe_XmasBranch_C' | 'Recipe_XmasBow_C' | 'Recipe_CandyCane_C' | 'Recipe_Biomass_Leaves_C' | 'Recipe_Biomass_Wood_C' | 'Recipe_IronPlateReinforced_C' | 'Recipe_Concrete_C' | 'Recipe_Screw_C' | 'Recipe_Cable_C' | 'Recipe_Wire_C' | 'Recipe_IngotCopper_C' | 'Recipe_Fireworks_01_C' | 'Recipe_Fireworks_02_C' | 'Recipe_Fireworks_03_C' | 'RecipeCustom_Build_GeneratorCoal_C_Desc_Coal_C' | 'RecipeCustom_Build_GeneratorCoal_C_Desc_CompactedCoal_C' | 'RecipeCustom_Build_GeneratorCoal_C_Desc_PetroleumCoke_C' | 'RecipeCustom_Build_GeneratorFuel_C_Desc_LiquidFuel_C' | 'RecipeCustom_Build_GeneratorFuel_C_Desc_LiquidTurboFuel_C' | 'RecipeCustom_Build_GeneratorFuel_C_Desc_LiquidBiofuel_C' | 'RecipeCustom_Build_GeneratorFuel_C_Desc_RocketFuel_C' | 'RecipeCustom_Build_GeneratorFuel_C_Desc_IonizedFuel_C' | 'RecipeCustom_Build_GeneratorBiomass_Automated_C_Desc_Leaves_C' | 'RecipeCustom_Build_GeneratorBiomass_Automated_C_Desc_Wood_C' | 'RecipeCustom_Build_GeneratorBiomass_Automated_C_Desc_Mycelia_C' | 'RecipeCustom_Build_GeneratorBiomass_Automated_C_Desc_GenericBiomass_C' | 'RecipeCustom_Build_GeneratorBiomass_Automated_C_Desc_Biofuel_C' | 'RecipeCustom_Build_GeneratorBiomass_Automated_C_Desc_PackagedBiofuel_C' | 'RecipeCustom_Build_GeneratorNuclear_C_Desc_NuclearFuelRod_C' | 'RecipeCustom_Build_GeneratorNuclear_C_Desc_PlutoniumFuelRod_C' | 'RecipeCustom_Build_GeneratorNuclear_C_Desc_FicsoniumFuelRod_C';
```

Comment:
Please keep this as-is; I'm one lining it to avoid messing around with deltas in PRs 😄 
Not sure if this already excluded in .prettierignore, could you add it in case? Thanks

Reply:
Fixed1